src/__tests__/TuxAnimations-test.js

```diff
@@ -35,42 +40,47 @@ describe('Animations', function () {
 
   it("applies a class of 'fly' for a fly transition", function () {
     ideaFlyComponent = TestUtils.renderIntoDocument(
-      <Fly key={mocks.ideas[0].id}>
-        <Idea key={mocks.ideas[0].id} />
-      </Fly>
+      <Fly key={mocks.ideas[0].id} />
     );
+    ideaFlyComponent.setProps({children: <Idea key={mocks.ideas[0].id} /> })
     var ideaFly = TestUtils.findRenderedDOMComponentWithClass(ideaFlyComponent, 'fly');
     expect(ideaFly).toBeDefined();
   });
 
-  it("applies a class of 'fade' for a fade transition", function () {
+   it("applies a class of 'fade' for a fade transition", function () {
     ideaFadeComponent = TestUtils.renderIntoDocument(
-      <Fade key={mocks.ideas[1].id}>
-        <Idea key={mocks.ideas[1].id} />
-      </Fade>
+      <Fade key={mocks.ideas[0].id} />
     );
+    ideaFadeComponent.setProps({children: <Idea key={mocks.ideas[0].id} /> })
     var ideaFade = TestUtils.findRenderedDOMComponentWithClass(ideaFadeComponent, 'fade');
     expect(ideaFade).toBeDefined();
   });
 
-  it("applies a class of 'zoom' for a zoom transition", function () {
+   it("applies a class of 'zoom' for a zoom transition", function () {
     ideaZoomComponent = TestUtils.renderIntoDocument(
-      <Zoom key={mocks.ideas[1].id}>
-        <Idea key={mocks.ideas[1].id} />
-      </Zoom>
+      <Zoom key={mocks.ideas[0].id} />
     );
+    ideaZoomComponent.setProps({children: <Idea key={mocks.ideas[0].id} /> })
```

Comment:
Explain why we need to use `setProps` here and why we are setting the `children` prop.  Remove whitespace between `>` and `}`.